test/dynamo/test_misc.py

```diff
@@ -2190,7 +2199,7 @@ def test_dynamo_min_operator_with_shape(self):
         def f(x, a):
             return min(x.shape[0], a)
 
-        result = f(torch.ones(6), 3)
+        result = f(torch.tensor([1, 1, 1, 1, 1, 1]), 3)
```

Comment:
Weird

Reply:
Oh was just testing stuff, this doesn't matter. I can change it back.

Reply:
rm